lib/tasks/attendance_csv_importer.rake

```diff
@@ -4,7 +4,9 @@ task attendance_csv_importer: :environment do
   # Pull single file from Wonderschool URL
   # Process single file through Attendance Processor
   # Archive file to S3
-  AttendanceCsvImporter.new.call
+  start_date = ENV['start_date']&.to_date || 1.year.before
```

Comment:
Can you talk to me about the choice to pass a static date or default to a dynamic time range here?

Reply:
I'm still not exactly sure about what the optimal date range is, so I wanted to provide the option to configure the range according to the center. I'm assuming that without params, setting the earliest date to a year ago is enough to catch all the changes. This is all assuming that this task is used on a regular basis with most recent dates in mind, if this task is more often used to onboard the entire attendance records of a single new center, I believe it might be better to just default to nil, so that the default behavior is to look at all records.

Reply:
Ahhhh gotcha!  So, for more clarity, this task is run every single day to pull data from Wonderschool.  Wonderschool provides us with a CSV link to *every attendance that has ever been entered* for all of the users in our system, and we pull that every day to get new attendance records.  What changes would you make based on that information?

Reply:
In that case I can default the start date value to the day before?

Reply:
If a year ago is doable without impacting performance, a year would be great. If not, one month back is good.

Reply:
@OliverCai0 I think a year is reasonable, shouldn't be too bad on performance and we can always change it later.  Can you set it to 1.year.ago from the date the task is run?